src/components/SubPage/convert-data.ts

```diff
@@ -0,0 +1,133 @@
+import { isEmpty } from "lodash";
+import {
+    Diagram,
+    Clone,
+    DiseaseCellLineNode,
+    Sequence,
+} from "../../component-queries/types";
+import { DiagramCardProps } from "../shared/DiagramCard";
+import {
+    ClonePercentPositive,
+    UnpackedDiseaseCellLineFull,
+    UnpackedEditingDesign,
+} from "./types";
+
+export const unpackDiagrams = (diagrams?: Diagram[]): DiagramCardProps[] => {
+    if (!diagrams || diagrams.length === 0) {
+        return [];
+    }
+    return diagrams.map((diagram) => {
+        if (!diagram.image) {
+            return {
+                title: diagram.title,
+                caption: diagram.caption,
+            };
+        }
+        return {
+            title: diagram.title,
+            caption: diagram.caption,
+            image: diagram.image.childImageSharp.gatsbyImageData,
+        };
```

Comment:
```suggestion
        return {
            title: diagram.title,
            caption: diagram.caption,
            image: diagram.image ? diagram.image.childImageSharp.gatsbyImageData : undefined,
        };
```

Check the syntax on this but I think you could just do one return statement?